components/Templates/includes/functions-view_template.php

```diff
@@ -297,30 +299,106 @@ function frontier_do_subtemplate( $atts, $content ) {
 				}
 
 				$out .= pods_shortcode( array(
-					'name'  => $pod->fields[ $atts[ 'field' ] ][ 'pick_val' ],
+					'name'  => $field[ 'pick_val' ],
 					'slug'  => $target_id,
 					'index' => $key
 				), $template );
 
 			}
-		} elseif ( 'file' == $pod->fields[ $atts[ 'field' ] ][ 'type' ] && 'attachment' == $pod->fields[ $atts[ 'field' ] ][ 'options' ][ 'file_uploader' ] && 'multi' == $pod->fields[ $atts[ 'field' ] ][ 'options' ][ 'file_format_type' ] ) {
+		} elseif ( 'file' == $field[ 'type' ] && 'attachment' == $field[ 'options' ][ 'file_uploader' ] && 'multi' == $field[ 'options' ][ 'file_format_type' ] ) {
 			$template = frontier_decode_template( $content, $atts );
 			foreach ( $entries as $key => $entry ) {
 				$content = str_replace( '{_index}', $key, $template );
 				$content = str_replace( '{@_img', '{@image_attachment.' . $entry[ 'ID' ], $content );
 				$content = str_replace( '{@_src', '{@image_attachment_url.' . $entry[ 'ID' ], $content );
-				$content = str_replace( '{@' . $atts[ 'field' ] . '}', '{@image_attachment.' . $entry[ 'ID' ] . '}', $content );
+				$content = str_replace( '{@' . $field_name . '}', '{@image_attachment.' . $entry[ 'ID' ] . '}', $content );
 
 				$out .= pods_do_shortcode( $pod->do_magic_tags( $content ), frontier_get_shortcodes() );
 			}
 
+		} else {
+			// Relationship to something other than a Pod (ie: user)
+			foreach ( $entries as $key => $entry ) {
+				$template = frontier_decode_template( $content, $atts );
+				$content = str_replace( '{_index}', $key, $template );
+
+				$out .= pods_do_shortcode( frontier_pseudo_magic_tags( $template, $entry, $pod ), frontier_get_shortcodes() );
+			}
 		}
 	}
 
 	return pods_do_shortcode( $out, frontier_get_shortcodes() );
 }
 
 /**
+ *
+ * Search and replace like Pods magic tags but with an array of data instead of a Pod
+ *
+ * @param Pod $pod
+ * @param string $template
+ * @param array $data
+ *
+ * @return string
+ */
```

Comment:
add @since 2.7. ?

Reply:
Good catch.  If this is going to get added we can add whatever version it is beforehand.